### PR TITLE
organization view and edit templates 

### PR DESCRIPTION
**Pull request by Yog**
#86

### CODE REVIEW

File: templates/volontulo/organization_form.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
```
Comment (filipgorczynski) [inline templates/volontulo/organization_form.html]: OK, but with #73 there is new 2 additional step: extending col1/col2 and those extends base.

File: urls.py [hunk 2]
```diff
@@ -30,4 +30,14 @@
         views.show_offer,
         name='show_offer'
     ),
+    url(
```
Comment (filipgorczynski) [inline urls.py:33]: I think here should be some [organization] ID to distinct between all organizations.

File: templates/volontulo/organization_form.html (hunk 1 above)
Comment (Yog) [inline templates/volontulo/organization_form.html]: it's going to be {% extends "common/col2.html" %} then

File: urls.py (hunk 2 above)
Comment (Yog) [inline urls.py:33]: that will be up to anyone who is going to programm it i think

**Pull request merged**